src/app/governance/page.tsx

```diff
@@ -69,6 +68,8 @@ function ProposalList() {
     };
   }, [proposals]);
 
+  const votingPower = useGovernanceVotingPower(address);
```

Comment:
nit: since you're not using the loading and/or the error return values, maybe 
```suggestion
  const { votingPower } = useGovernanceVotingPower(address);
```

to make the following code neater

Reply:
dobra